src/heist_path/HeistPathB.py

```
@@ -484,16 +503,85 @@ def lg2_wp2_to_exit2_safer(self):
+                _key = self._current_fighter_key or _key
+            self.send_key(_key)
+            self.next_frame()
+            if not self._find_red_health_bar(10):
+                if settle < 0:
+                    settle = time.time()
+                if time.time() - settle > 2:
+                    break
+            else:
+                settle = -1
+        else:
+            raise ("timeout for clear_current_combat")
```

Ruff (0.15.14)

[warning] Cannot raise a literal. Did you intend to return it or raise an Exception?

(B016)